doc/develop/dataio.rst

```diff
@@ -104,6 +105,48 @@ To read data from an OME-TIFF file, one could write:
    print(das[0].metadata[model.MD_LENS_MAG])
    print(das[0].metadata)
 
+Image position metadata
+=======================
+Most of the data store correspond to "spatial" data, i.e., an image representing
+the with axes in X and Y. In order to ensure a perfect overlay between data from
+different acquisition type, the metadata describes precisely how the image should
+be positioned. The following metadata has influence:
+
+* ``MD_PIXEL_SIZE``: size of a pixel (in m) in X and Y. In other words, this is the scale.
+* ``MD_POS``: the position (in m) of the *center of the image* in X and Y
+* ``MD_ROTATION``: counter-clockwise rotation (in radians) applied to the image from its center
+* ``MD_SHEAR``: *vertical* shear
+
+All values default to 0, excepted for the ``MD_PIXEL_SIZE`` which is required.
+
+Note that during acquisition, all these metadata have a twin-brother named with an extra
+`_COR` in order to record the correction on the image display. The function ``util.img.mergeMetadata()``
+can be used to merge these corrections into the main metadata. Before saving data
+to a file, the correction is automatically merged. So typically, after opening a
+file, the data will not have any of these extra correction metadata.
+
+For a given pixel situated at coordinates *p = i, j* in an image, its "physical" position
+*P = x, y* (in meters) can be computed with the following formula *P = RSLp + T*,
+where R is the rotation, L is the scale, and S is the shear.
+
+In Python, this can be done with:
+
+.. code-block:: python
+
+   from odemis import model
+   from odemis.util.transform import AffineTransform
+   pxs = md.get[model.MD_PIXEL_SIZE]
+   translation = md.get(model.MD_POS, (0, 0))
+   rotation = md.get(model.MD_ROTATION, 0)
+   shear = md.get(model.MD_SHEAR, 0)
+
+   tform = AffineTransform(rotation=rotation, scale=pxs, translation=translation)
+   L = numpy.array([(1, 0), (-shear, 1)])
+   tform.transformation_matrix = numpy.dot(tform.transformation_matrix, L)
+
+   P = tform(p)
+   p = tform.inverse()(P)
```

Comment:
I decided to try this code, and I thought small p would stay equal. However it doesn't:

```
tform = AffineTransform(rotation=1.2, scale=2, translation=2)
P = tform(((1, 1), (2, 2)))
p = tform.inverse()(P)
p
 
 ([[1.93203909, -0.50172042],
 [4.22643593,  1.06796091]])
```

I expected it to return ((1, 1), (2, 2)).

Reply:
Translation should be 2 floats, not a single value. If you use (2,2), or (2,3), that code works.
That said, I've found extra issues in the example code, and I'll update it.

Also, I don't understand what should mean translation = 2, but I would expect that either it does the same as `(2,2)`, as the "worst acceptable behaviour", and not cause the inversion to break. Anyway, I think it should just raise a ValueError.

Reply:
Indeed, I meant to do `translation = (2, 2)`. And I would also expect `translation=2` to do the same as `(2, 2)` or raise an error. 

The same could actually be said for scale. The docstring says scale is the scale in x and in y. However it doesn't raise an error when you pass a float, it will then use it as `(2, 2)`.

Reply:
I'll add the type checks in that code.

Reply:
I'll send a separate PR for checking the arguments of the transformations.